newIDE/app/src/Utils/GDevelopServices/Authentication.js

```diff
@@ -141,8 +144,25 @@ export default class Authentication {
 
   getFirebaseUser = (cb: (any, ?FirebaseUser) => void) => {
     if (!this.isAuthenticated()) return cb({ unauthenticated: true });
+    this.auth.currentUser.reload().then(() => cb(null, this.firebaseUser));
+  };
 
-    cb(null, this.firebaseUser);
+  sendFirebaseEmailVerification = (cb: () => void): Promise<void> => {
+    return this.auth.currentUser.reload().then(() => {
+      if (this.firebaseUser && !this.firebaseUser.emailVerified) {
```

Comment:
ubernitpicking: I tend to early return when you want to dismiss a "not happy path". This saves one indentation and allow to reason like "if this case happens, just bail out".

```suggestion
      if (!this.firebaseUser || this.firebaseUser.emailVerified) return;
```

This reads fairly clearly: "if you don't have a user or it's verified, just stop here".
And you can continue reading the code without remembering "ok there is maybe a else/else if later in the code".